### PR TITLE
DOCSP-41017 Remove support for Debian 10, RHEL 7.0, and Amazon 2

### PR DESCRIPTION
**Pull request by pierwill**
DOCSP-41017
### Self-Review Checklist

- [ ] Check that the submodule pulled in the right changes (if applicable).
- [ ] [Define](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions) taxonomy [values](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) at top of page.
- [ ] Add genre facets (tutorial or reference), as in this [example PR](https://github.com/10gen/cloud-docs/pull/5042).
- [ ] Add programmingLanguage (if necessary).
- [ ] Add meta keywords (if necessary).
- [ ] Resolve any new warnings or errors in the build.
- [ ] Proofread for spelling and grammatical errors.
- [ ] Check staging for rendering issues.
- [ ] Confirm links are working.

### CODE REVIEW

File: source/install-atlas-cli.txt [hunk 1]
```diff
@@ -143,32 +143,6 @@ Follow These Steps
 
                   .. tabs::
 
-                     .. tab:: RHEL
```
Comment (gssbzn) [inline source/install-atlas-cli.txt:146]: other versions of rhel are still supported, tho I'm not sure these are the same instructions

**Review by gssbzn — COMMENTED** [review]
please update https://www.mongodb.com/docs/atlas/cli/current/compatibility/

**Comment by sarahsimpers** [comment]
We need to remove Amazon Linux 2 and Debian 10 from the Enterprise tab as well

**Review by sarahsimpers — APPROVED** [review]
LGTM

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**